src/app/tests/TestReadInteraction.cpp

```diff
@@ -67,11 +67,17 @@ chip::EndpointId kInvalidTestEndpointId = 3;
 chip::DataVersion kTestDataVersion1     = 3;
 chip::DataVersion kTestDataVersion2     = 5;
 
+static chip::System::Clock::Internal::MockClock mockClock;
+static chip::System::Clock::ClockBase * realClock;
```

Comment:
rename mockClock to gMockClock , realClock to gRealClock, will make ti more obvious.